exhibitor-core/src/main/java/com/netflix/exhibitor/core/azure/AzureClientImpl.java

```diff
@@ -0,0 +1,63 @@
+package com.netflix.exhibitor.core.azure;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.blob.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+
+public class AzureClientImpl implements AzureClient {
+    private final Logger log = LoggerFactory.getLogger(getClass());
+    private final String storageConnectionString;
+
+    public AzureClientImpl(AzureCredential credentials) {
+        this.storageConnectionString =
+            "DefaultEndpointsProtocol=http" +
```

Comment:
Apparently, this used StringBuilder under the covers:
http://stackoverflow.com/a/8725765